inference-engine/samples/speech_sample/main.cpp

```diff
@@ -631,8 +652,18 @@ int main(int argc, char *argv[]) {
                                   &numFrames,
                                   &numFrameElements,
                                   &numBytesPerElement);
-                auto floatScaleFactor =
-                        ScaleFactorForQuantization(ptrFeatures.data(), MAX_VAL_2B_FEAT, numFrames * numFrameElements);
+                float floatScaleFactor;
+                if (!FLAGS_iname.empty() && (std::find(input_uses.begin(), input_uses.end(), all_inputs[i]) == input_uses.end())) {
+                    // If the input isn't used, its scale factor changes to 0
+                    floatScaleFactor = 0.0f;
+                    if (numFrames > 1) {
+                        throw std::logic_error("Should be 1 frame for infer");
```

Comment:
Could you please add more detailed explanation in error message here? As far as I understand - if inputs names are used in should be 1 frame.